corehq/apps/data_interfaces/tests/test_forms.py

```diff
@@ -0,0 +1,48 @@
+import json
+from unittest.mock import patch
+from django.test import SimpleTestCase
+from corehq.apps.data_interfaces.forms import CaseRuleCriteriaForm
+from corehq.apps.data_interfaces import forms
+
+
+class CaseRuleCriteriaFormTests(SimpleTestCase):
+    def test_validation_with_fully_specified_location(self):
+        post_data = self._create_form_input(location_filter={
+            'name': 'TestLocation',
+            'location_id': '123',
+            'include_child_locations': 'false',
```

Comment:
Is this intended to be a boolean value? When this is dumped to JSON it will become `"false"` (a string), which doesn't seem right.

Reply:
This is meant to mimic what our javascript sends over, which is the string 'true' or 'false'. That will ultimately be converted  to a boolean using [this code](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_interfaces/forms.py#L43-L49)